common.md

```diff
@@ -163,15 +163,27 @@ represent mutez overflow.
   rule #IsLegalMutezValue(I) => I >=Int 0 andBool I <Int #MutezOverflowLimit
 ```
 
-Michelson byte literals are given by their hexadecimal representation with the
-prefix "Ox". Since the K byte literal has a different representation, we convert
-here from one to the other.
+Michelson byte literals are represented internally by the K `Bytes` type.
 
 ```k
-  syntax MBytes ::= Bytes
+  syntax MichelsonBytes ::= Bytes
 
-  syntax Bytes ::= #MBytesLiteralToBytes(MBytesLiteral) [function, hook(BYTES.hexstring2bytes)]
-  rule `MBytesLiteral`(M) => #MBytesLiteralToBytes(M)
+  syntax String ::= #MichelsonBytesTokenToString(MichelsonBytesToken) [function, hook(STRING.token2string)]
+  // ------------------------------------------------------------------------------------------------------
+  rule `MichelsonBytesToken`(M) => #ParseBytes(stripFirst(#MichelsonBytesTokenToString(M), 2), .Bytes)
```

Comment:
```suggestion
  syntax String ::= #MichelsonBytesTokenToString(MichelsonBytesToken) [function, hook(STRING.token2string)]

  rule `MichelsonBytesToken`(M) => #ParseBytes(stripFirst(#MichelsonBytesTokenToString(M), 2), .Bytes)
```

This comment is confusing. To me it implies that the following line is the implementation of the hook.